### PR TITLE
Prevent keyboard interrupt errors

### PR DESCRIPTION
**Pull request by pranavbaburaj**
- Prevent inputs from capturing keyboard interrupts.
- Replaced `try..except` block with `os.path.isfile(filename)` for validating files

 - [x] Join the Piston CLI Discord Community?
 - [x] If dependencies have been added or updated, run pipenv lock?
 - [x] Lint your code (pipenv run lint)?
 - [ ] Set the PR to allow edits from contributors?

### CODE REVIEW

**Comment by pranavbaburaj** [comment]
Closes #62

**Review by Shivansh-007 — CHANGES_REQUESTED** [review]
Two comments here,

1. Firstly you need to open a issue first, get it approved and assigned, then you can start working on the issue. Sometimes we don't approve issues so your PR could go waste.

2. Secondly, I am not 100% convinced with this solution, I had rather have it [`piston/__main__.py`](https://github.com/Piston-CLI/piston-cli/blob/main/piston/__main__.py). So that it handles all the errors together, it isn't "pythonic" to go put a `try..except` everywhere this can happen (there are few places where this still occurs, like the code input in `from_input.py`)

**Comment by Shivansh-007** [comment]
Better commit messages please.

A in-depth guide to writing great commit messages can be found in Chris Beam's [_How to Write a Git Commit Message_](https://chris.beams.io/posts/git-commit/).

**Comment by pranavbaburaj** [comment]
> Better commit messages please. (pointing to [d37063e](https://github.com/Piston-CLI/piston-cli/commit/d37063e72fc2a18611c495af35ce2b70858e97da))
> 
> A in-depth guide to writing great commit messages can be found in Chris Beam's [_How to Write a Git Commit Message_](https://chris.beams.io/posts/git-commit/).

I am sorry for that 😐

File: piston/utils/helpers.py [hunk 1]
```diff
@@ -29,6 +29,15 @@ def close() -> None:
...
 
+def input_prompt(text: str) -> str:
+    """Prompt the user."""
+    try:
+        user_input = CONSOLE.input(text)
+        return user_input
+    except KeyboardInterrupt:
+        close()
+
+
```
Comment (pranavbaburaj) [inline piston/utils/helpers.py:40]: This is the function that handles the input

**Pull request closed without merging**